repose-aggregator/components/filters/versioning-filter/src/test/java/org/openrepose/filters/versioning/util/ContentTransformerTest.java

```diff
@@ -36,19 +35,14 @@
 import javax.xml.bind.JAXBContext;
 import javax.xml.bind.JAXBElement;
 import javax.xml.bind.JAXBException;
-import java.io.ByteArrayOutputStream;
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.UncheckedIOException;
+import java.io.*;
 import java.net.URISyntaxException;
 import java.util.Map;
 
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.CoreMatchers.not;
-import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.CoreMatchers.*;
```

Comment:
`CoreMatchers` instead of just `Matchers`? As I understand it, `Matchers` is a superset containing the matchers in `CoreMatchers`. I don't think we make the effort to use the minimal set, so I wonder if it wouldn't be simpler to just import `Matchers` everywhere we need them.